flyway/gxa/migrations/V9__gxa-create-exp-design.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE gxa_marker_gene
+(
+    experiment_accession VARCHAR(255) CHECK (experiment_accession LIKE 'E-%') NOT NULL,
+    assay VARCHAR(255) NOT NULL,
+    assay_id VARCHAR(255) NOT NULL,
+    gene_id VARCHAR(255) NOT NULL,
+    gene_name VARCHAR(255) NOT NULL,
+    gene_rank INTEGER,
+    expression_level FLOAT NOT NULL,
+    expression_unit VARCHAR(8) NOT NULL,
+    sample_number INTEGER NOT NULL,
```

Comment:
```suggestion
    number_assays INTEGER NOT NULL,
```
I'm not sure about this.  `sample_number` is not accurate